CharlieBackend.Api.UnitTes/LessonServiceTests.cs

```diff
@@ -95,7 +95,8 @@ public async Task CeateLessonAsync()
                 );
 
             //Act
-            var result = await lessonService.CreateLessonAsync(createLessonDto);
+            var Result = await lessonService.CreateLessonAsync(createLessonDto);
+            var result = Result.Data;
```

Comment:
Do you run tests? Does everything work for these changes?

Reply:
all properties should be in lowerCamelCase and two same names make it vary hard to understand

Reply:
yep